### PR TITLE
Replace ntfs-3g_ntfsprogs with Linux kernel 5.15 ntfs3

### PR DESCRIPTION
**Pull request by heitbaum**
### DONE ✅
- Dependant on #5738 
- RPi now updated #5888
- AMlogic now updated #5920
  - Amlogic kernel update/image/package.mk
  - **Note: Amlogic is already configured and compiling ntfs3 in the kernel**
- update build environment to swap out fuse.ntfs entirely
  - packages/virtual
  - projects/Amlogic/options  - config/show_config
  - packages/sysutils/ntfs-3g_ntfsprogs
- updated udevil to include mount.ntfs and fix default ntfs3 options
  - https://github.com/IgnorantGuru/udevil/issues/
  - https://github.com/IgnorantGuru/udevil/issues/103
  - https://github.com/alpharde/udevil/commits/master
- tested
- bugs (upstream ntfs3 patches have been included)
  - included in default `patches` - **ntfs3: patches since 5.15** 1f983f2193c8f86c6782b5bd4047c709ed586871
  - https://github.com/Paragon-Software-Group/linux-ntfs3/tree/52e00ea6b26e45fb8159e3b57cdde8d3f9bdd8e9
  - https://github.com/torvalds/linux/compare/master...Paragon-Software-Group:master
 
## TODO
- include ntfs3: patches since 5.15 in `amlogic` and `raspberrypi` ?

### Waiting on Paragon / others
- Paragon: 
  - `Yes, we plan to publish and open-source our mkfs.NTFS utility.`
  - tools: (mkfs.ntfs)
  - `We may also open-source rapidcopyfile and backup utilities for NTFS once mkfs has been published.` 
  - https://www.paragon-software.com/home/ntfs3-driver-faq/
- ntfs3.fsck - this still required Windows

~~### OUTSTANDING BUGS~~
- ~~https://bugzilla.kernel.org/show_bug.cgi?id=214833~~
- ~~https://www.spinics.net/lists/ntfs3/msg00630.html~~

mail list - https://www.spinics.net/lists/ntfs3/index.html

### CODE REVIEW

File: packages/sysutils/udevil/package.mk [hunk 1]
```diff
@@ -26,6 +26,12 @@ post_makeinstall_target() {
 
   mkdir -p ${INSTALL}/usr/bin
     cp -PR src/udevil ${INSTALL}/usr/bin
+
+  mkdir -p ${INSTALL}/usr/sbin
```
Comment (CvH) [inline packages/sysutils/udevil/package.mk:30]: why is this needed for ntfs and not for ext4 etc ?

Comment (heitbaum) [reply packages/sysutils/udevil/package.mk:30]: The -t doesn’t match (ntfs != ntfs3) and udevil checks /proc/filesystems - https://github.com/IgnorantGuru/udevil/issues/103 it is definitely not elegant. Would be happy for a better idea.

**Comment by CvH** [comment]
@heitbaum pls rebase

**Comment by heitbaum** [comment]
> @heitbaum pls rebase

Rebased 👍

**Review by CvH — APPROVED** [review]
